build.sbt

```diff
@@ -36,7 +36,7 @@ lazy val aggregateCompile = ScopeFilter(
 lazy val commonSettings = Seq(
   organization         := "org.zalando",
   name                 := "grafter",
-  version in ThisBuild := "1.4.3"
+  version in ThisBuild := "1.4.6"
```

Comment:
Yes, I don't know what happened here but the last version is indeed [`1.4.5`](https://oss.sonatype.org/content/repositories/releases/org/zalando/grafter_2.12).